From a50e6e03c49b6708c762ffbed86a18266ec6e698 Mon Sep 17 00:00:00 2001
From: Manuel Herrmann <0@0x17.de>
Date: Sat, 4 Aug 2018 17:47:26 +0200
Subject: [PATCH] indentation adjustments

---
 google-c-style.el | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/google-c-style.el b/google-c-style.el
index 9bb12c6..5087f20 100644
--- a/google-c-style.el
+++ b/google-c-style.el
@@ -68,7 +68,7 @@ Suitable for inclusion in `c-offsets-alist'."
 (defconst google-c-style
   `((c-recognize-knr-p . nil)
     (c-enable-xemacs-performance-kludge-p . t) ; speed up indentation in XEmacs
-    (c-basic-offset . 2)
+    (c-basic-offset . 4)
     (indent-tabs-mode . nil)
     (c-comment-only-line-offset . 0)
     (c-hanging-braces-alist . ((defun-open after)
@@ -107,7 +107,7 @@ Suitable for inclusion in `c-offsets-alist'."
                        scope-operator))
     (c-offsets-alist . ((arglist-intro google-c-lineup-expression-plus-4)
                         (func-decl-cont . ++)
-                        (member-init-intro . ++)
+                        (member-init-intro . +)
                         (inher-intro . ++)
                         (comment-intro . 0)
                         (arglist-close . c-lineup-arglist)
@@ -126,7 +126,7 @@ Suitable for inclusion in `c-offsets-alist'."
                         (case-label . +)
                         (statement-case-open . +)
                         (statement-case-intro . +) ; case w/o {
-                        (access-label . /)
+                        (access-label . -)
                         (innamespace . 0))))
   "Google C/C++ Programming Style.")
 
-- 
2.16.1

